components/addresses/index.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable react/no-unused-state */
+import React, { Component } from 'react';
+import { View } from 'react-native';
+import SearchBar from '../searchBar';
+import SearchBarDestination from '../searchBarDestination';
+import BackButton from './backButton';
+import CurrentLocation from './currentLocation';
+import Destination from './destination';
+import Car from './car';
+import Bus from './bus';
+import Walking from './walking';
+import Bike from './bike';
+import styles from './styles';
+
+export default class Addresses extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      region: {
+        latitude: 0,
+        longitude: 0,
+        latitudeDelta: 0.05,
+        longitudeDelta: 0.05
+      },
+      hide: true,
+    };
+  }
+
+    updateRegion = (newRegion) => {
+      this.setState({
+        region: {
+          latitude: newRegion.latitude,
+          longitude: newRegion.longitude,
+          latitudeDelta: 0.05,
+          longitudeDelta: 0.05
+        }
+      });
+      this.props.getRegion(newRegion);
+    };
+
+
+    updateCoordinates = (newCoordinates) => {
+      this.setState({
+        coordinates: newCoordinates
+      });
+      this.props.getCoordinates(newCoordinates);
+    };
+
+    getPolylinePoint = (data) => {
+      this.setState({
+        encryptedLine: data
+      });
+    };
+
+    render() {
+      return (
+        <View style={styles.searchContainer}>
+          <SearchBar updateRegion={this.updateRegion} hideMenu={this.state.hide} />
```

Comment:
There is a weird behaviour here where the first search bar has a back button when I click on it.
![IMG_0011](https://user-images.githubusercontent.com/30513164/76670476-0c435e80-6567-11ea-8ae2-fed8c1c220fe.PNG)

Reply:
so does the second search bar

Reply:
both search bar has back buttons, it's part of the search bar element that was implemented from 'react-native-elements'
it doesn't go back to the first page, it's similar to the in the android OS.

Reply:
I understand. It is good then.